lib/gutenberg/gutenberg-editor-sidebar-component.js

```diff
@@ -62,4 +62,11 @@ export default class GutenbergEditorSidebarComponent extends AsyncBaseContainer
 			}
 		);
 	}
+
+	async trashPost() {
+		const trashSelector = By.css( 'button.editor-post-trash' );
+
+		await driverHelper.waitTillPresentAndDisplayed( this.driver, trashSelector );
```

Comment:
Have you tried without this line? I think it will work fine without it, that `clickWhenClickable` is enough.

Reply:
Yeah, it is really there just to give a little more time for the autosave to kick in so that the trash button shows up. It was there in the original so I kept it.